### ISSUE TITLE
Add url extract feature

### ISSUE THREAD

**Issue by mesb1** [iawia002/annie#764]
Hi.

Can we have some kind of -g feature as youtube-dl does?

I would like annie extract direct url for video and stdout it with desired quality.
youtube-dl -f 22 -g http://addr
Provides me with direct utl to 720p video, that i can use as where i want.
Basically i want use annie as url extractor for direct urls from youtube..

**Comment by MegaCron** [comment]
I agree with mesb1, the function of getting a direct link is needed.
